functions.py

```diff
@@ -2,39 +2,22 @@
 from PIL import Image
 import os, sys
 
+class ImageResizer:
 
-def cut_image():
-    print '''
-    =====================
-    CUT IMAGE
-    =====================
-    1. File
-    2. Folder
-    3. Close
-    =====================
-    '''
-    choice = raw_input(">> ")
-    if choice == "1":
-        directory = raw_input('Directory to file: ')
-        width = input('Width: ')
-        height = input('Height: ')
-        name = raw_input('Save as: ')
+    def cut_image(self, directory, width, height, name):
         im = Image.open(directory)
         im = im.resize((width, height))
         if not os.path.exists('resized_images'):
             os.mkdir('resized_images')
         try:
-            print "Size: " + str(width) + " x " + str(height)
             im.save('resized_images/'+name)
             print "- " + name + " : " + "DONE"
```

Comment:
And i forget one important thing :) **ALL** `print` instructions from this class go to `Main` class to a proper place :)